backend/model/db.js

```diff
@@ -13,6 +13,8 @@ const TitleModel = require('./titles')
 const TagModel = require('./tags')
 const UniversityModel = require('./universities')
 const DepartmentModel = require('./departments')
+const NotificationModel = require('./notification')
```

Comment:
all other models have plural file names, should be consistent

Reply:
yeah sure, @FarukOzderim of course they are not important, but they just occured to me and wanted to say it. and since for others you will edit the code anyway, I thought you could spare 5 seconds to also resolve this.